library/core/src/primitive_docs.rs

```diff
@@ -0,0 +1,1294 @@
+// `library/{std,core}/src/primitive_docs.rs` should have the same contents.
```

Comment:
This content comes from somewhere but I don't see the equivalent removed content. Is it normal?

Reply:
I don't understand the question. This is duplicated from `library/std/src/primitive_docs.rs`, like the comment says.

Reply:
This was my question somehow: why is it duplicated and not simply moved?

Reply:
Other question: if you need to have a file at this location, why not making a link instead?

Reply:
See https://github.com/rust-lang/rust/pull/87073#issuecomment-903155679 and https://rust-lang.zulipchat.com/#narrow/stream/122651-general/topic/Who.20should.20review.20changes.20to.20linkchecker.3F.

Reply:
@GuillaumeGomez are you ok with this hack?

Reply:
Not really but I couldn't find something better. The improvements compensate more than enough for this though. So let's go for it!